app/javascript/react/components/BrandShowContainer.js

```diff
@@ -29,19 +32,44 @@ const BrandShowContainer = (props) => {
       getPizzas()
   }, [])
 
+
   const pizzaTiles = brand.pizzas.map( (pizza) => {
     return(
       <BrandShowTile name={pizza.product_name} id={pizza.id} key={pizza.id} />
     )
   })
-  
+
+  const addPizza = async (formPayload) => {
+    const response = await fetch("/api/v1/pizzas/", {
```

Comment:
fetch requests should be wrapped in a `try..catch` block in case there are any errors that arise!